script/seed_bucket_data.py

```diff
@@ -9,21 +9,27 @@
     fs.copy(
```

Comment:
Is this still needed/usable? Just wondering if it's worth adding a comment if it doesn't align any more (sorry I realize it's just getting picked up because of linter stuff, but given that it's here....)

Reply:
Good question; I added a comment saying it was deprecated, for the same reason as the other thing